qiskit/visualization/state_visualization.py

```diff
@@ -280,15 +276,15 @@ def plot_bloch_multivector(rho, title='', figsize=None):
     return fig
 
 
-def plot_state_city(rho, title="", figsize=None, color=None,
+def plot_state_city(state, title="", figsize=None, color=None,
                     alpha=1, ax_real=None, ax_imag=None):
```

Comment:
```suggestion
@deprecate_arguments({'rho': 'state'})
def plot_state_city(state, title="", figsize=None, color=None,
                     alpha=1, ax_real=None, ax_imag=None, *, rho=None):  # pylint: unused-argument 
```
with
`from qiskit.util import deprecate_arguments`